src/js/modules/core/components/PageLayout.js

```diff
@@ -14,24 +18,61 @@ const styles = {
   }
 };
 
-const PageLayout = ({ classes, children, location, topLevelSectionsWithDirectChildren }) => {
+const sidebarStyles = {
+  sidebar: {
+    background: '#fff',
+    boxShadow: '1px 2px 12px 0 rgba(0, 0, 0, 0.5)',
+    padding: '14px',
+    width: '220px',
+    zIndex: 5001,
+  },
+  overlay: {
+    background: 'rgba(255, 255, 255, 0.8)',
+    zIndex: 5000,
+  }
+}
+
+const PageLayout = ({
+                      classes,
+                      children,
+                      location,
+                      isSidebarOpen,
+                      openSidebar,
+                      closeSidebar,
+                    }) => {
+  const handleSetSidebar = shouldSetSidebarOpen => {
```

Comment:
shouldSetSidebarOpen? Just make it isSidebarOpen. Also how about handleSetOpen or handleSidebar